### PR TITLE
[Develop] Fave 17952 affix

### PR DESCRIPTION
**Pull request by Kelvin-GohKengBoon**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

# Screenshots (if applicable)

Please add appropriate screenshots.

### CODE REVIEW

File: src/Affix/stories/index.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import { default as Affix } from '../index'
+export default {
+  title: 'Affix',
+  component: Affix,
+} as ComponentMeta<typeof Affix>;
+
+
+var indents = [];
+for (var i = 0; i < 5; i++) {
+  indents.push(<div>Only for Testing Purpose (Scroll Down)</div>);
```
Comment (naimdanialjohar) [inline src/Affix/stories/index.stories.tsx:14]: provide a `key` to the div with the value of the `i`

File: src/Affix/stories/index.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+  return(
+    <div style={{ height:"600vh" }}>
+      <Affix offsetTop={0}>
+        <div style={notificationStyle}>
+          <span style={{ alignSelf:"center" }}>Notification (Affix Top Right)</span>
+        </div>
+      </Affix>
+      {indents}
+      {indents}
```
Comment (naimdanialjohar) [inline src/Affix/stories/index.stories.tsx:59]: one `indents` should be fine. if you want to double it, i suggest doubling the value of `i` in the `for loop` above

File: src/Affix/stories/index.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        <Alert
+          message={message}
+          type={type}
+          showIcon={false}
+          closable={false}
+          banner={false}
+        />
+      </Affix>
+      {indents}
+      {indents}
```
Comment (naimdanialjohar) [inline src/Affix/stories/index.stories.tsx:82]: same comment as above

File: src/Affix/stories/index.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+  return(
+    <div style={{ height:"600vh" }}>
+      <Affix offsetTop={0}>
+        <div style={navStyle}>
+          <span style={{ alignSelf:"center" }}>Nav Bar (Affix Top)</span>
+        </div>
+      </Affix>
+      {indents}
+      {indents}
```
Comment (naimdanialjohar) [inline src/Affix/stories/index.stories.tsx:34]: remove one `indents` here

**Review by naimdanialjohar — APPROVED** [review]

**Review by AladdinNullptr — APPROVED** [review]

**Pull request merged**